source/includes/library-reference/_brain.html.md

```diff
@@ -13,73 +13,54 @@ such that ">>" indicates a decreasing order of precedence. Note that failure to
 BRAIN name in at least one of these places will result in a friendly error.
 
 ## Brain(config, name)
-Creates a local object for interacting with an existing BRAIN on the server.
-
-```cpp
-auto config = make_shared<bonsai::Config>(argc, argv);
-auto brain = make_shared<bonsai::Brain>(config);
-std::cout << brain << std::endl;
-```
 
 ```python
 config = bonsai_ai.Config(sys.argv)
 brain = bonsai_ai.Brain(config)
 print(brain)
 ```
 
+Creates a local object for interacting with an existing BRAIN on the server.
+
 | Argument | Description |
 | ---      | ---         |
-| `config` | Object returned by previously created `Bonsai::Config` (C++) or `bonsai_ai.Config()` (Python). |
+| `config` | Object returned by previously created `bonsai_ai.Config()`. |
 | `name`   | BRAIN name as specified on the server. If name is empty, the BRAIN name in `config` is used instead. |
 
 ## update()
-```cpp
-brain.update();
-```
 
 ```python
 brain.update()
 ```
+
 Refreshes description, status, and other information with the current state of the BRAIN on the server.
 Called by default when constructing a new Brain object.
 
-## string name()
-```cpp
-std::cout << brain.name() << endl;
-```
+## name()
 
 ```python
 print(brain.name)
```

Comment:
Here and elsewhere, this doesn't quite match the pattern for the rest of the library, which is more like `brain.name == "my brain"`. Also, these aren't actually accessor functions; in Python, we access these fields directly, so the headers shouldn't include the trailing parens.

Reply:
Alternatively, we could probably condense these into a table, which might be a little more readable. There's not a lot of content in the descriptions, anyway, and the code snippets are already meaningless. Food for thought.